index.js

```diff
@@ -57,10 +57,11 @@ const inputs = [
 readSequential(inputs)
     .then(options => {
         return getCertificate(options).then(certs => {
+            if(!certs) return Promise.reject("Could not get certs. Check previous errors");
```

Comment:
This was a sidefect of the catching I did on lib.js#112. Reverting that this is not needed. thanks for spotting it.